helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -210,8 +211,9 @@ internal fun OrtResult.getProcessedCopyrightStatements(
             if (omitExcluded && pathExcludes.isNotEmpty()) return@innerForEach
 
             val statements = processor
-                .process(licenseFindings.copyrights.map { it.statement })
+                .process(licenseFindings.copyrights.map { it.statement }.filterNot { it in copyrightGarbage })
                 .toMutableSet()
+                .filterNot { it in copyrightGarbage }
```

Comment:
Can the processor really create new garbage? In the `NoticeReporter` we only filter garbage before passing the copyrights to the processor, but not afterwards.

Reply:
I think it can: as a processed statement is not guaranteed to be equal to the unprocessed statement the processed statement might be on the garbage list while the unprocessed is not. I think as is is the right way to do it. Let me file a commit for the `NoticeReporter` to fix that.

Reply:
Updated also the `NoticeReporter`.

Reply:
Thanks, but this conflicts with my heavy notice reporter refactoring, I would not really want to rebase on top of that, because I move a lot of code around and add another notice reporter which would need the same change. Would you mind removing those two commits again and I apply them again after my PR is merged?

Reply:
ok, done.

Reply:
Thanks.